src/ragger/firmware/fatstacks/positions.py

```diff
@@ -233,6 +233,7 @@ def __iter__(self):
             "next": BUTTON_LOWER_RIGHT,
         },
         "UseCaseAddressConfirmation": {
+            "tap": BUTTON_ABOVE_LOWER_MIDDLE,
```

Comment:
so what's the difference with 'confirm' ?

Reply:
There is no difference except that the "confirm" action is meant for the "Confirm" button, and the "tap" action is meant for the "Tap to continue" action.
I think that having both help understand what is supposed to be done when they are in an navigate_and_compare instruction list for example.

Reply:
Ok, I see. It is less confusing with a dedicated instruction indeed.

Reply:
So in order to keep meaningful names, could it be "continue" or is it too restrictive?

Reply:
I've named it "Tap" to be equivalent with the already existing `USE_CASE_REVIEW_TAP`